pkgs/applications/networking/irc/chatterino2/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, pkgconfig, fetchFromGitHub, qtbase, qtsvg, qtmultimedia, qmake, boost, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "chatterino2";
+  version = "2.0.4";
+  name = "${pname}-${version}";
+  src = fetchFromGitHub {
+    owner = "fourtf";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1vdln1wfdd1s1prb9iab5spyfv34jvq7p8cwc3313izhbq5c2r05";
+    fetchSubmodules = true;
+  };
+  nativeBuildInputs = [ qmake pkgconfig ];
+  buildInputs = [ qtbase qtsvg qtmultimedia boost openssl ];
+  installPhase = ''
```

Comment:
Does the `.pro` file included not take care of installing the binary automatically? If not I'd suggest that might be worth a PR upstream...

Reply:
Fixed upstream in the relevant branch, `make install` now installs all relevant stuff to /usr/local (modifiable with `qmake PREFIX=/usr`)